service/src/main/java/bio/terra/workspace/app/controller/ControlledResourceControllerBase.java

```diff
@@ -34,14 +37,24 @@ public ControlledResourceControllerBase(
   public ControlledResourceFields toCommonFields(
       UUID workspaceUuid,
       ApiControlledResourceCommonFields apiCommonFields,
-      AuthenticatedUserRequest userRequest) {
+      String region,
+      AuthenticatedUserRequest userRequest,
+      WsmResourceType wsmResourceType) {
 
     ManagedByType managedBy = ManagedByType.fromApi(apiCommonFields.getManagedBy());
     AccessScopeType accessScopeType = AccessScopeType.fromApi(apiCommonFields.getAccessScope());
     PrivateUserRole privateUserRole =
         computePrivateUserRole(workspaceUuid, apiCommonFields, userRequest);
 
     String userEmail = getSamService().getUserEmailFromSamAndRethrowOnInterrupt(userRequest);
+    if (wsmResourceType != WsmResourceType.CONTROLLED_AZURE_STORAGE_CONTAINER
```

Comment:
Nitpick: Might be good to extract these to a static set we can compare against rather than a long-ish combo of &&s here.